Sources/Moya/MoyaProvider.swift

```diff
@@ -119,10 +119,15 @@ open class MoyaProvider<Target: TargetType>: MoyaProviderType {
     open func request(_ target: Target,
                       callbackQueue: DispatchQueue? = .none,
                       progress: ProgressBlock? = .none,
+                      priority: Float = URLSessionTask.defaultPriority,
                       completion: @escaping Completion) -> Cancellable {
 
         let callbackQueue = callbackQueue ?? self.callbackQueue
-        return requestNormal(target, callbackQueue: callbackQueue, progress: progress, completion: completion)
+        return requestNormal(target, callbackQueue: callbackQueue, progress: progress, priority: priority, completion: completion)
+    }
+
+    public func request(_ target: Target, callbackQueue: DispatchQueue?, progress: ProgressBlock?, completion: @escaping Completion) -> Cancellable {
```

Comment:
why this additional function? 

does the function above with the new `priority` param with a default value not maintain API compatibility?

Reply:
@AndrewSB Old C++ habit ! I fixed it.